assets-src/styles/sass/30-base/_tables.scss

```diff
@@ -60,6 +64,8 @@ https://adrianroselli.com/2020/11/under-engineered-responsive-tables.html
 
         table {
             border: 0;
+			margin-bottom: 0;
+			margin-top: 0;
```

Comment:
Same comment: code indentation needs fixing.

Reply:
Please re-check after pulling latest commits. I'm not sure if my default IDE indent spacing has changed between spaces and tabs.